app/controllers/post-settings-menu.js

```diff
@@ -290,39 +290,39 @@ export default Controller.extend(SettingsMenuMixin, {
          * Action sent by post settings menu view.
          * (#1351)
          */
-        setPublishedAt(userInput) {
+        setPublishedAtUTC(userInput) {
             if (!userInput) {
-                // Clear out the publishedAt field for a draft
+                // Clear out the publishedAtUTC field for a draft
                 if (this.get('model.isDraft')) {
-                    this.set('model.publishedAt', null);
+                    this.set('model.publishedAtUTC', null);
                 }
                 return;
             }
 
             // The user inputs a date which he expects to be in his timezone. Therefore, from now on
             // we have to work with the timezone offset which we get from the timeZone Service.
             this.get('timeZone.blogTimezone').then((blogTimezone) => {
-                let newPublishedAt = parseDateString(userInput, blogTimezone);
-                let publishedAt = moment.utc(this.get('model.publishedAt'));
+                let newPublishedAtUTC = parseDateString(userInput, blogTimezone);
```

Comment:
This may be a little confusing as we get date from `parseDateString` that is actually in the blog's timezone, not in UTC. We actually convert it into UTC further down on line 319.

I would suggest two variables here, one that holds the blog-timezone date and one that holds the UTC value.